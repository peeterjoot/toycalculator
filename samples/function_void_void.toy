FUNCTION bar ( )
{
    PRINT "In bar";
    RETURN;
};

FUNCTION foo ( )
{
    INT16 v;
    v = 3;
    PRINT "In foo";
    CALL bar();
    //CALL bar( v );
    PRINT "Called bar";
    RETURN;
};

PRINT "In main";
CALL foo();
PRINT "Back in main";
