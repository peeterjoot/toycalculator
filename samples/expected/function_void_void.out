In main
In foo
In bar
Called bar
Back in main
